spec/forms/claim/provider_form_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+describe Claim::ProviderForm, type: :model do
+  let!(:school) { create(:claims_school) }
+  let!(:provider1) { create(:provider, name: "Best Practice Network") }
```

Comment:
Given you might need that setup a lot, is it worth have a trait in the factory for add a provider of that name?

Reply:
Yeah, that's a good idea